src/index.js

```diff
@@ -183,9 +219,16 @@ export default class Firedux {
   }
   logout () {
     const { dispatch } = this
+
     return new Promise((resolve, reject) => {
       dispatch({type: 'FIREBASE_LOGOUT_ATTEMPT'})
-      this.ref.unauth()
+      if (this.v3) {
+        this.auth().signOut()
+        .then(() => resolve(), (error) => reject(error))
+      } else {
+        this.ref.unauth() // no callbacks for old firebase :(
```

Comment:
> remove the dist builds from the PR or regenerate for a clean merge?
I got it.

anyway, i'm bit confused about testing. i've tried to add the test case for logout to make sure working properly, however i got an error which is
> Your API key is invalid, please check you have copied it correctly.

should i run the npm like this?
`npm test FIREBASE_API_KEY=something api key`
or just put the apiKey in ref.js?